src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -672,20 +653,17 @@ protected VariantFilter makeVariantFilter() {
      */
     private SortedSet<String> createSampleNameInclusionList(Map<String, VCFHeader> vcfHeaders) {
         final SortedSet<String> vcfSamples = VcfUtils.getSortedSampleSet(vcfHeaders, GATKVariantContextUtils.GenotypeMergeType.REQUIRE_UNIQUE);
-        final Collection<String> samplesFromFile = SampleUtils.getSamplesFromFiles(sampleFiles);
         final Collection<String> samplesFromExpressions = matchSamplesExpressions(vcfSamples, sampleExpressions);
 
         // first, check overlap between requested and present samples
-        final Set<String> commandLineUniqueSamples = new LinkedHashSet<>(samplesFromFile.size()+samplesFromExpressions.size()+sampleNames.size());
-        commandLineUniqueSamples.addAll(samplesFromFile);
+        final Set<String> commandLineUniqueSamples = new LinkedHashSet<>(samplesFromExpressions.size()+sampleNames.size());
```

Comment:
Might be good to add a comment that this `commandLineUniqueSamples` Set is only used for error message output.